js/piwik.js

```diff
@@ -3799,8 +3802,11 @@ if (typeof window.Piwik !== 'object') {
                     // we only reuse visitorId when used on same device / browser
 
                     var currentTimestampInSeconds = getCurrentTimestampInSeconds();
-                    var timeoutInSeconds = 45;
+                    var timeoutInSeconds = configVisitorIdUrlParameterTimeoutInSeconds;
 
+                    if (timeoutInSeconds <= 0) {
+                        return true;
+                    }
                     if (currentTimestampInSeconds >= timestampInUrl
                         && currentTimestampInSeconds <= (timestampInUrl + timeoutInSeconds)) {
                         // we only use visitorId if it was generated max 45 seconds ago
```

Comment:
comment is outdated, as it's not 45 seconds, but `configVisitorIdUrlParameterTimeoutInSeconds` instead